cli/src/diff_util.rs

```diff
@@ -730,8 +730,8 @@ enum DiffLineType {
 }
 
 struct UnifiedDiffHunk<'content> {
-    left_line_range: Range<usize>,
-    right_line_range: Range<usize>,
+    removed_line_range: Range<usize>,
+    added_line_range: Range<usize>,
```

Comment:
I think the old names were clearer because the left/right ranges are not all removed/added. In the case of context lines, you can have a left/right line that's not removed or added.

Reply:
Good point. Dropped the rename patch.